Crate nimiq-transaction doesn't build with the feature serde-derive

The crate `nimiq-transaction` doesn't build with the feature `serde-derive`. This is the output of building the crate with the feature `serde-derive`:
```
2/2: serde-derive
error[E0433]: failed to resolve: could not find `Serialize` in `serde`
   --> primitives/transaction/src/account/htlc_contract.rs:166:52
    |
166 | #[cfg_attr(feature = "serde-derive", derive(serde::Serialize, serde::Deserialize))]
    |                                                    ^^^^^^^^^ could not find `Serialize` in `serde`

error[E0433]: failed to resolve: could not find `Deserialize` in `serde`
   --> primitives/transaction/src/account/htlc_contract.rs:166:70
    |
166 | #[cfg_attr(feature = "serde-derive", derive(serde::Serialize, serde::Deserialize))]
    |                                                                      ^^^^^^^^^^^ could not find `Deserialize` in `serde`

error[E0433]: failed to resolve: could not find `Serialize` in `serde`
   --> primitives/transaction/src/account/htlc_contract.rs:196:52
    |
196 | #[cfg_attr(feature = "serde-derive", derive(serde::Serialize, serde::Deserialize))]
    |                                                    ^^^^^^^^^ could not find `Serialize` in `serde`

error[E0433]: failed to resolve: could not find `Deserialize` in `serde`
   --> primitives/transaction/src/account/htlc_contract.rs:196:70
    |
196 | #[cfg_attr(feature = "serde-derive", derive(serde::Serialize, serde::Deserialize))]
    |                                                                      ^^^^^^^^^^^ could not find `Deserialize` in `serde`

error[E0433]: failed to resolve: could not find `Serialize` in `serde`
  --> primitives/transaction/src/account/staking_contract/structs.rs:69:52
   |
69 | #[cfg_attr(feature = "serde-derive", derive(serde::Serialize, serde::Deserialize))]
   |                                                    ^^^^^^^^^ could not find `Serialize` in `serde`

error[E0433]: failed to resolve: could not find `Deserialize` in `serde`
  --> primitives/transaction/src/account/staking_contract/structs.rs:69:70
   |
69 | #[cfg_attr(feature = "serde-derive", derive(serde::Serialize, serde::Deserialize))]
   |                                                                      ^^^^^^^^^^^ could not find `Deserialize` in `serde`

error[E0433]: failed to resolve: could not find `Serialize` in `serde`
  --> primitives/transaction/src/account/vesting_contract.rs:81:52
   |
81 | #[cfg_attr(feature = "serde-derive", derive(serde::Serialize, serde::Deserialize))]
   |                                                    ^^^^^^^^^ could not find `Serialize` in `serde`

error[E0433]: failed to resolve: could not find `Deserialize` in `serde`
  --> primitives/transaction/src/account/vesting_contract.rs:81:70
   |
81 | #[cfg_attr(feature = "serde-derive", derive(serde::Serialize, serde::Deserialize))]
   |                                                                      ^^^^^^^^^^^ could not find `Deserialize` in `serde`

error[E0433]: failed to resolve: could not find `Serialize` in `serde`
  --> primitives/transaction/src/lib.rs:61:56
   |
61 |     #[cfg_attr(feature = "serde-derive", derive(serde::Serialize, serde::Deserialize), serde(try_from = "u8", into = "u8"))]
   |                                                        ^^^^^^^^^ could not find `Serialize` in `serde`

error[E0433]: failed to resolve: could not find `Deserialize` in `serde`
  --> primitives/transaction/src/lib.rs:61:74
   |
61 |     #[cfg_attr(feature = "serde-derive", derive(serde::Serialize, serde::Deserialize), serde(try_from = "u8", into = "u8"))]
   |                                                                          ^^^^^^^^^^^ could not find `Deserialize` in `serde`

error: cannot find attribute `serde` in this scope
  --> primitives/transaction/src/account/staking_contract/structs.rs:78:46
   |
78 |         #[cfg_attr(feature = "serde-derive", serde(skip))]
   |                                              ^^^^^

error: cannot find attribute `serde` in this scope
  --> primitives/transaction/src/account/staking_contract/structs.rs:88:46
   |
88 |         #[cfg_attr(feature = "serde-derive", serde(skip))]
   |                                              ^^^^^

error: cannot find attribute `serde` in this scope
  --> primitives/transaction/src/account/staking_contract/structs.rs:94:46
   |
94 |         #[cfg_attr(feature = "serde-derive", serde(skip))]
   |                                              ^^^^^

error: cannot find attribute `serde` in this scope
   --> primitives/transaction/src/account/staking_contract/structs.rs:100:46
    |
100 |         #[cfg_attr(feature = "serde-derive", serde(skip))]
    |                                              ^^^^^

error: cannot find attribute `serde` in this scope
   --> primitives/transaction/src/account/staking_contract/structs.rs:106:46
    |
106 |         #[cfg_attr(feature = "serde-derive", serde(skip))]
    |                                              ^^^^^

error: cannot find attribute `serde` in this scope
   --> primitives/transaction/src/account/staking_contract/structs.rs:111:46
    |
111 |         #[cfg_attr(feature = "serde-derive", serde(skip))]
    |                                              ^^^^^

error: cannot find attribute `serde` in this scope
   --> primitives/transaction/src/account/staking_contract/structs.rs:119:46
    |
119 |         #[cfg_attr(feature = "serde-derive", serde(skip))]
    |                                              ^^^^^

error: cannot find attribute `serde` in this scope
   --> primitives/transaction/src/account/staking_contract/structs.rs:124:46
    |
124 |         #[cfg_attr(feature = "serde-derive", serde(skip))]
    |                                              ^^^^^

error: cannot find attribute `serde` in this scope
   --> primitives/transaction/src/account/staking_contract/structs.rs:129:46
    |
129 |         #[cfg_attr(feature = "serde-derive", serde(skip))]
    |                                              ^^^^^

error: cannot find attribute `serde` in this scope
  --> primitives/transaction/src/lib.rs:61:88
   |
61 |     #[cfg_attr(feature = "serde-derive", derive(serde::Serialize, serde::Deserialize), serde(try_from = "u8", into = "u8"))]
   |                                                                                        ^^^^^

For more information about this error, try `rustc --explain E0433`.
error: could not compile `nimiq-transaction` due to 20 previous errors
Failed to build feature serde-derive for package at primitives/transaction/

```

--- a/primitives/Cargo.toml
+++ b/primitives/Cargo.toml
@@ -27,7 +27,7 @@ num-bigint = { version = "0.2", optional = true }
 num-traits = { version = "0.2", optional = true }
 parking_lot = { version = "0.9", optional = true }
 regex = { version = "1.3", optional = true }
-serde = { version = "1.0", optional = true }
+serde = { version = "1.0", features = ["derive"], optional = true }
 strum_macros = "0.20"
 
 beserial = { path = "../beserial" }

--- a/primitives/account/Cargo.toml
+++ b/primitives/account/Cargo.toml
@@ -16,7 +16,7 @@ lazy_static = "1.3"
 log = "0.4"
 parking_lot = "0.9"
 rand = "0.7"
-serde = { version = "1.0", optional = true }
+serde = { version = "1.0", features = ["derive"], optional = true }
 strum_macros = "0.20"
 thiserror = "1.0"
 
@@ -26,9 +26,9 @@ nimiq-bls = { path = "../../bls" }
 nimiq-collections = { path = "../../collections", features = ["bitset"] }
 nimiq-database = { path = "../../database" }
 nimiq-hash = { path = "../../hash" }
-nimiq-keys = { path = "../../keys" }
-nimiq-primitives = { path = "..", features = ["coin", "policy", "slots"] }
-nimiq-transaction = { path = "../transaction" }
+nimiq-keys = { path = "../../keys", features = ["serde-derive"] }
+nimiq-primitives = { path = "..", features = ["coin", "policy", "serde-derive", "slots"] }
+nimiq-transaction = { path = "../transaction", features = ["serde-derive"] }
 nimiq-trie = { path = "../trie" }
 nimiq-utils = { path = "../../utils", features = ["hash-rng"] }
 nimiq-vrf = { path = "../../vrf" }

--- a/primitives/transaction/Cargo.toml
+++ b/primitives/transaction/Cargo.toml
@@ -21,11 +21,11 @@ thiserror = "1.0"
 
 beserial = { path = "../../beserial" }
 beserial_derive = { path = "../../beserial/beserial_derive" }
-nimiq-bls = { path = "../../bls", features = ["beserial"] }
-nimiq-hash = { path = "../../hash" }
-nimiq-keys = { path = "../../keys" }
+nimiq-bls = { path = "../../bls", features = ["serde-derive"] }
+nimiq-hash = { path = "../../hash", features = ["serde-derive"] }
+nimiq-keys = { path = "../../keys", features = ["serde-derive"] }
 nimiq-macros = { path = "../../macros" }
-nimiq-primitives = { path = "..", features = ["policy", "networks", "account", "coin"] }
+nimiq-primitives = { path = "..", features = ["account", "coin", "networks", "policy", "serde-derive"] }
 nimiq-utils = { path = "../../utils", features = ["merkle"] }
 
 [dev-dependencies]

